components/sysutils/mbuffer/Makefile

```diff
@@ -13,32 +13,38 @@
 # Copyright 2011 EveryCity Ltd. All rights reserved.
 # Copyright 2013, Jon Tibble.  All rights reserved.
 # Copyright 2016, Jim Klimov
+# Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
 #
+
+BUILD_BITS= 64
+
 include ../../../make-rules/shared-macros.mk
 
+GCC_VERSION= 10
+
```

Comment:
I wonder if we should make gcc 10 our new default instead of manually set it all over.

Reply:
If is assured that the compilation with gcc-10 works without problems it can removed from the Makefile.